### DISCUSSION TITLE
Pi 4 Error PC Mining is disabled

### DISCUSSION THREAD

**Discussion by emathews** [revoxhere/duino-coin#641] [General]
Was working until yesterday and now this, not sure what to do here

![2021-07-07-091402_2560x1440_scrot](https://user-images.githubusercontent.com/934351/124768597-cc944900-df06-11eb-878e-c7fff59e1ba2.png)

**Comment by revoxhere** [comment]
You are running an outdated version

**Reply by emathews** [reply]
thank you!
